deploy/kind/cleanup.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# ==== Config (override via env) ====
+CLUSTER="${CLUSTER:-kind-moonlink-dev}"
+NS="${NS:-moonlink}"
+MANIFEST_DIR="${MANIFEST_DIR:-deploy/kind}"
```

Comment:
So it looks like a requirement that by default setup/cleanup scripts should run under project root directory, worth mentioning in the readme